StudentTimeScedulerV2/src/main/java/za/ac/cput/studenttimescedulerv2/Settings.java

```diff
@@ -0,0 +1,131 @@
+package com.example.checkbox;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.*;
+import javafx.scene.layout.*;
+import javafx.collections.*;
+import javafx.stage.Stage;
+import javafx.scene.paint.*;
+
+public class Settings extends Application {
+
+    // launch the application
+    public void start(Stage s)
```

Comment:
Comment out from here pls. I have addressed this issue in video and voice notes. The application already have a main method. GUI stuff should be in a separate file. I will accept this for now but it needs to be commented out to avoid the project from breaking